Rakefile

```diff
@@ -11,7 +11,7 @@ end
 
 task :default => :spec
 
-require 'rake/rdoctask'
+require 'rdoc/task'
```

Comment:
This whole task was removed here: https://github.com/mgomes/api_auth/commit/54e1a90f9aae0f3859c430a5d65e15773a7c9182